pkg/keyspace/tso_keyspace_group.go

```diff
@@ -146,6 +148,47 @@ func (m *GroupManager) Close() {
 	m.wg.Wait()
 }
 
+func (m *GroupManager) allocDefaultNodesForKeyspaceGroup() {
+	defer logutil.LogPanic()
+	defer m.wg.Done()
+	ticker := time.NewTicker(allocNodesForDefaultKeyspaceGroupInterval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-m.ctx.Done():
+			return
+		case <-ticker.C:
+		}
+		countOfNodes := m.GetNodesCount()
+		if countOfNodes < utils.KeyspaceGroupDefaultReplicaCount {
+			log.Info("the count of nodes is not enough to allocate the default keyspace group", zap.Int("count", countOfNodes))
```

Comment:
Will this log keep flushing every second?

Reply:
I will adjust it